takwimu/static/js/charts.js

```diff
@@ -466,10 +465,10 @@ function Chart(options) {
     chart.htmlBase = chart.chartContainer
       .append("div")
       .attr("class", "column-set")
-      .style("margin-top", function() {
-        return chart.chartChartShowYAxis ? -chart.settings.height + "px" : "0";
-      })
-      .style("height", chart.settings.height + "px");
+      .style("margin-top", 0)
```

Comment:
Can't we just add our configuration **if** `chartChartShowYAxis` is `true` rather than _assuming_ it's always true. This way, we can reuse this file in HURUmap at the end of this sprint and make this option available to all HURUmap apps.